plutus-chain-index/src/Plutus/ChainIndex/UtxoState.hs

```diff
@@ -96,14 +103,15 @@ fromTx tx =
         , _tubUnmatchedSpentInputs = Set.mapMonotonic txInRef (view citxInputs tx)
         }
 
-type UtxoIndex a = FingerTree (UtxoState a) (UtxoState a)
-instance Monoid a => Measured (UtxoState a) (UtxoState a) where
-    measure = id
+type UtxoIndex a = FingerTree (Sum Int, UtxoState a) (UtxoState a)
```

Comment:
We could move the block count to `UtxoState` and give it a name

Reply:
Or define `newtype BlockCount = BlockCount Int deriving (Semigroup, Monoid) via (Sum Int)`  or something